graders/c/cgrader/cgrader.py

```diff
@@ -63,7 +63,7 @@
 """
 
 
-class UngradableException(Exception):
+class UngradableError(Exception):
```

Comment:
We can't make breaking changes like this in our public-facing API.

Reply:
Reverted in cb0dd63df

Reply:
I would be curious if there are any usages however.

Reply:
There definitely are, I checked!

Reply:
One option is to rename it and to create a new "deprecated" alias to it.